sensor/kubernetes/sensor/sensor.go

```diff
@@ -110,6 +111,8 @@ func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 	complianceMultiplexer := compliance.NewMultiplexer()
 	// TODO(ROX-16931): Turn auditLogEventsInput and auditLogCollectionManager into ComplianceComponents if possible
 	complianceService := compliance.NewService(o, auditLogEventsInput, auditLogCollectionManager, complianceMultiplexer.ComplianceC())
+	var collectorC chan *sensorInternal.MsgToCollector
+	collectorRuntimeConfigService := collector.NewService(collectorC)
```

Comment:
I am missing a `make` and a `close` for this channel. Usually, we follow the approach that the go routine that writes to that channel - there must be exactly one - opens and also closes the channel. This is a pattern to avoid panics when writing to a closed channel.

Reply:
There is now one `make` and one `close`. Both are in `service_impl`.